content/ja/docs/contribute/review/reviewing-prs.md

```diff
@@ -31,13 +31,44 @@ weight: 10
 
 ## レビューのプロセス
 
-一般に、コンテンツや文体に対するプルリクエストは、英語でレビューを行います。
+一般に、コンテンツや文体に対するプルリクエストは、英語でレビューを行います。図1は、レビュープロセスについて手順の概要を示しています。 各ステップの詳細は次のとおりです。
+
+<!-- See https://github.com/kubernetes/website/issues/28808 for live-editor URL to this figure -->
+<!-- You can also cut/paste the mermaid code into the live editor at https://mermaid-js.github.io/mermaid-live-editor to play around with it -->
+
+{{< mermaid >}}
+flowchart LR
```

Comment:
We can read localized text on other language(e.g. [zh-cn](https://deploy-preview-39722--kubernetes-io-main-staging.netlify.app/zh-cn/docs/contribute/review/reviewing-prs/#review-process)).
Can you translate here to Japanese?

Reply:
I also translated the inside of mermaid.